<?xml version="1.0"?>
<launch>
  <arg name="manager_name" default="nodelet_manager" />

  <node pkg="nodelet" type="nodelet" name="$(arg manager_name)" args="manager" output="screen">
  </node>

  <node pkg="nodelet" type="nodelet" name="mask_rcnn_nodelet"
      args="load camera_apps/MaskRcnnNodelet $(arg manager_name)" >
  <param name="model_path" value="$(find camera_apps)/model/mask_rcnn" />
    <rosparam file="$(find camera_apps)/config/param/mask_rcnn.yaml" command="load"/>
  </node>    
  <node pkg="rviz" name="rviz" type="rviz" args="-d $(find camera_apps)/config/rviz/camera_apps.rviz" output="screen"/>

  <node name="image_republish" pkg="image_transport" type="republish" args="compressed raw">
      <remap from="in" to="/camera/color/image_raw/" />
      <remap from="out" to="/camera/color/image_raw" />
  </node>

</launch>
